tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -1079,12 +1104,14 @@ HRESULT CorProfiler::TryRejitModule(ModuleID module_id)
         // We call the function to analyze the module and request the ReJIT of integrations defined in this module.
         if (tracer_integration_preprocessor != nullptr && !integration_definitions_.empty())
         {
-            const auto numReJITs = tracer_integration_preprocessor->RequestRejitForLoadedModules(std::vector<ModuleID>{module_id}, integration_definitions_);
+            const auto numReJITs = tracer_integration_preprocessor->RequestRejitForLoadedModules(
+                std::vector<ModuleID>{module_id}, integration_definitions_);
             Logger::Debug("[Tracer] Total number of ReJIT Requested: ", numReJITs);
         }
     }
 
-    return S_OK;
+    return
+        S_OK;
```

Comment:
Silly reformatter

Reply:
argh, tried to play with settings but to no avail.. fixed in b9b2c8c95e0dc5c966184933b404f5d1ed7d06ba